torch/quantization/_correct_bias.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import print_function, division, absolute_import
+import torch
+import torch.nn as nn
+import torch.nn.quantized as nnq
+
+import torch.quantization
+import torch.quantization._numeric_suite as ns
+
+_supported_modules = {nn.Linear, nn.Conv2d}
+_supported_modules_quantized = {nnq.Linear, nnq.Conv2d}
+
+def get_module(model, name):
+    ''' Given name of submodule, this function grabs the submodule from given model
+    '''
+    return {name: submodule for name, submodule in model.named_modules()}[name]
```

Comment:
you can just do model.named_modules()[name]

Reply:
model.named_modules() returns a generator not a dictionary, you can't index into it with a key
i can flip this to 
for module_name, submodule in model.named_modules():
    if name == module_name:
         return submodule
return None

Reply:
hmm, maybe call `dict(model.named_modules())`?